src/chains/ethereum/transaction/src/eip1559-fee-market-transaction.ts

```diff
@@ -115,7 +116,7 @@ export class EIP1559FeeMarketTransaction extends RuntimeTransaction {
       blockNumber: this.blockNumber ? this.blockNumber : null,
       transactionIndex: this.index ? this.index : null,
       from: this.from,
-      to: this.to.isNull() ? null : this.to,
+      to: this.to ? this.to : null,
```

Comment:
Can this be simplified to:

```suggestion
      to: this.to,
```

?

Reply:
(same in the other transaction types that do this same check)